tokenomics/sagittarius-engine.md

```diff
@@ -1,32 +1,52 @@
 # Sagittarius Engine
 
 ## Introduction
-The Sagittarius Engine adds new two new utilities to the MKR token. MKR can be used to 
-- Farm SubDAO tokens
-- Generate Dai using MKR as collateral
+The Sagittarius Engine is a module that incentivizes MKR holders to lock their MKR tokens up for long periods of time. The locked-up MKR is required to participate in Maker Governance via delegation. 
 
-{% hint style="warning" %} This documentation describes planned functionality and processes that MakerDAO has not yet implemented. Be aware that parts may be inaccurate or out of date. {% endhint %}
+The incentives for locking up MKR in the Sagittarius Engine are:
+- Token farming
+- Dai generation through vaults with MKR as collateral
 
-## Usage
-Users may deposit MKR into the Sagittarius Engine. When depositing, they automatically delegate the deposited MKR through the Protocol Delegation System (PDS). Depositors must specify a target delegate smart contract through the PDS when making their deposit.
+Withdrawing MKR from the Sagittarius Engine burns part of the deposited MKR.
```

Comment:
My (old, possibly out-of-date) understanding was that this was only if you didn't wait the required lock-up period before exiting?

Reply:
The latest post doesn't have a lockup period. It reads as if 15% of the deposited MKR is burned upon exiting, regardless of when you do it.

Reply:
Can be merged if you're happy with this point about the lockup period.